# request
int32 id
---
# response
bool success